README.md

```diff
@@ -114,6 +115,7 @@ require("cmake-tools").setup {
         direction = "float", -- 'vertical' | 'horizontal' | 'tab' | 'float'
         close_on_exit = false, -- whether close the terminal when exit
         auto_scroll = true, -- whether auto scroll to the bottom
+        single_terminal_per_instance = true, -- single instance, autocloses the opened one, if present
```

Comment:
Maybe `singleton` is enough, what do you think?

Reply:
Sounds good to be, I was trying to be consistent with the terminal's settings